test/auth.test.ts

```diff
@@ -63,6 +63,27 @@ describe("auth api", () => {
     expect(data?.tokenCreate?.errors).not.toHaveLength(0);
   });
 
+  it("refreshes the auth token", async () => {
+    await saleor.auth.login({
+      email: TEST_AUTH_EMAIL as string,
+      password: TEST_AUTH_PASSWORD as string,
```

Comment:
To avoid having to cast them `as string`, default them to an empty string in the config file instead:
```
export const API_URI = process.env.API_URI || "";
export const TEST_AUTH_EMAIL = process.env.TEST_AUTH_EMAIL || "";
export const TEST_AUTH_PASSWORD = process.env.TEST_AUTH_PASSWORD || "";
```